src/auth/auth.module.ts

```diff
@@ -0,0 +1,26 @@
+import { Module } from '@nestjs/common';
+import { AuthController } from './auth.controller';
+import { AuthService } from './auth.service';
+import { UserModule } from '../user/user.module';
+import { JwtModule } from '@nestjs/jwt';
+import { jwtConstants } from './constants';
+import { LocalStrategy } from './strategies/local.strategy';
+import { JwtStrategy } from './strategies/jwt.strategy';
+import { PassportModule } from '@nestjs/passport';
+import { LocalAuthGuard } from './guards/local-auth.guard';
+
+@Module({
+  imports: [
+    UserModule,
+    PassportModule,
+    JwtModule.register({
+      global: true,
+      secret: jwtConstants.secret,
+      signOptions: { expiresIn: '30000s' },
```

Comment:
Let's say for now it's ok, but later we should generate a pair of tokens: access token and a refresh token. Then access token will be a short lived token - e.g. 5-30 minutes and refresh token will have a longer expiry date - e.g. 24h / 7 days / 30 days